adapters/http-vertx-base/pom.xml

```diff
@@ -29,4 +29,22 @@
     </dependency>
   </dependencies>
 
+  <build>
+    <plugins>
+      <plugin>
+        <groupId>org.jboss.jandex</groupId>
+        <artifactId>jandex-maven-plugin</artifactId>
```

Comment:
what is this good for?

Reply:
@sophokles73 As far as I understood, it is used to index beans across maven modules, so that they can be injected

This doc have some more info: https://quarkus.io/guides/cdi-reference